packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -38,7 +38,9 @@
     "react-use": "{{versionQuery 'react-use' '17.2.4'}}"
   },
   "peerDependencies": {
-    "react": "{{versionQuery 'react' '^16.13.1 || ^17.0.0 || ^18.0.0'}}"
+    "react": "{{versionQuery 'react' '^16.13.1 || ^17.0.0 || ^18.0.0'}}",
+    "react-dom": "{{versionQuery 'react-dom' '^16.13.1 || ^17.0.0 || ^18.0.0'}}",
```

Comment:
Hmm, not sure we necessarily want these as peers deps, we don't use them directly right? any strong opinion here?

Reply:
@Rugvip, after reading your comment, I went and took a second look at the error message. Actually, the problem starts from `/community-plugins/workspaces/<plugin-name-here>/node_modules/@backstage/app-defaults/dist'
node_modules/@backstage/app-defaults/dist/index.esm.js`. So, after inspecting the `@backstage/app-defaults` package.json I've found that this package lists these very same peerDependencies in [its own package.json](https://github.com/backstage/backstage/blob/7183fe7df9165b4b3147f09ded2c7cb4382e17cd/packages/app-defaults/package.json#L54). Now it is clear to me that adding the peerDependencies again is irrelevant. A better question IMO is: What package should be tasked with installing the required peerDeps that @backstage/app-defaults needs? Is it the brand-new frontend plugin or should it be performed at a higher hierarchy in the repo? WDYT?

In the meantime, I'll remove the peerDependencies but I'll leave only the devDependencies section. Please let me know if that's an acceptable solution 🙏